config/config.yml

```diff
@@ -0,0 +1,28 @@
+##
+## Example config file Replace fields as required
```

Comment:
I think this file should be removed, it is a duplicate of `example_config.yml` but is not as up to date